ext/soap/soap.c

```diff
@@ -4342,6 +4345,16 @@ static void delete_url(void *handle) /* {{{ */
 }
 /* }}} */
 
+static void delete_argv(struct _soap_class *class)
+{
+	if (class->argc) {
+		for (int i = 0; i < class->argc;i++) {
```

Comment:
nit: just for the style space before the incrementation :)

Reply:
Indeed, and for master this should even become uint32_t, which I'll do next...